### PR TITLE
[CI:DOCS] Include manifest example usage

### LINKED ISSUE #8872: Create Documentation for How to Use Manifests
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

Need a set of docs focused on creating image manifests and cross platform images. I believe most of the work is done here:

[mailing list with steps for using manifsts and making them work](https://lists.podman.io/archives/list/podman@lists.podman.io/thread/IOEAGRAIRPADSTSHE2O2TXROLCBABBZT/)

**Steps to reproduce the issue:**

1. Go to podman.io, click on search, and type "manifest" in the search box. 

**Describe the results you received:**

The only thing that comes up is  man pages. The man pages don't have all of the extra detail of a tutorial nor good examples. The man pages would augment this proposed documentation.

**Describe the results you expected:**
User should find a nice tutorial based off of the the steps described here:

[mailing list with steps for using manifsts and making them work](https://lists.podman.io/archives/list/podman@lists.podman.io/thread/IOEAGRAIRPADSTSHE2O2TXROLCBABBZT/)

**Additional information you deem important (e.g. issue happens only occasionally):**
I contemplated making this a RHEL documentation request, but I think it would be better upstream. Upstream first is the right approach for this.

### PR DESCRIPTION
**Pull request by cevich**
#### What this PR does / why we need it:

Add example usage and notes for the `podman manifest` subcommands.    I can make a similar update to the `buildah manifest` documentation once this is fine-tuned and merged.

#### How to verify it

Documentation will build successfully.  Build output must be manually reviewed by subject-matter experts.

#### Which issue(s) this PR fixes:

Fixes #8872

#### Special notes for your reviewer:

See also, the [podman.io blog post on this topic](https://podman.io/blogs/2021/10/11/multiarch.html).

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM
@flouthoc or @nalind PTAL

File: docs/source/markdown/podman-manifest.1.md [hunk 1]
```diff
@@ -24,5 +24,49 @@ The `podman manifest` command provides subcommands which can be used to:
...
+### Removing and tagging a manifest list before pushing
+
+Special care is needed when removing and pushing manifest lists, as opposed
+to the contents.  You almost always want to use the `manifest rm` and
+`manifest push --all` subcommands.  For example, a rename and push could
+be performed like this:
+
+        $ podman tag localhost/shazam example.com/example/shazam
+        $ podman manifest rm localhost/shazam
+        $ podman manifest push --all example.com/example/shazam
```
[the remark this answers is not in the document]
Comment (cevich) [reply docs/source/markdown/podman-manifest.1.md:68]: Yes, I've also questioned this and I'm certainly in support of a change.  I really don't see much/any use-case for removing individual bits/pieces of a manifest, when it's referenced by name.

I'd even extend that to `push --all`, what possible (non-evil) reason would someone have for pushing some but not all the contents?

[the remark this answers is not in the document]
Comment (nalind) [reply docs/source/markdown/podman-manifest.1.md:68]: I'm afraid I have no insight to offer on that one.

[the remark this answers is not in the document]
Comment (flouthoc) [reply docs/source/markdown/podman-manifest.1.md:68]: @cevich I think the similar process is being shared by docker as well for `rm` https://docs.docker.com/engine/reference/commandline/manifest_rm/ , I am in favor of hooking both manifest and images to `buildah/podman rm` but the problem is behavior would become significantly different from `docker` and we will loose parity.

[the remark this answers is not in the document]
Comment (flouthoc) [reply docs/source/markdown/podman-manifest.1.md:68]: @cevich Sorry I think i might misinterpreted the problem, could you please rephrase the problem which we have with `manifest rm`

**Pull request merged**